v-next/hardhat-utils/src/sub-process.ts

```diff
@@ -0,0 +1,27 @@
+import { spawn } from "node:child_process";
+
+/**
+ * Spawns a detached subprocess to execute a given file with optional arguments.
+ *
+ * @param absolutePathToSubProcessFile - The absolute path to the subprocess file to be executed.
+ * @param [args=[]] - Optional arguments to pass to the subprocess.
+ *
+ * The subprocess runs in a detached mode and its standard input/output is ignored.
+ * This function does not wait for the subprocess to complete and the subprocess is unreferenced
+ * to allow the parent process to exit independently.
+ */
+export function spawnSubProcess(
```

Comment:
should we handle errors?

Reply:
Good question. Not sure because if something fails is not related to hardhat errors. So not sure if it makes sense to catch anything. @alcuadrado what do you think?

Reply:
No, the subprocess should be responsible to handle its own errors

Reply:
If launching it fails, that'd be a fatal error and hardhat should crash.